PAWS/PAWSStandard.swift

```diff
@@ -231,10 +232,7 @@ actor PAWSStandard: Standard, EnvironmentAccessible, HealthKitConstraint, Onboar
         guard let accountStorage else {
             preconditionFailure("Account Storage was requested although not enabled in current configuration.")
         }
-        if let dob = details.dateOfBrith {
-            // Store whether the participant is older or younger than 18.
-            try await userDocumentReference.getDocument().setValue(dob.isAdultDateOfBirth, forKey: "ageGroupIsAdult")
-        }
+        // Store whether the participant is older or younger than 18.
```

Comment:
We should remove the comment here.

Nevertheless, on a higher level: Could actually always compute this and never need to store a flag if we store when the user has enrolled in the study within the user document that is generated when the user signs up (and where we currently store the invitation code). 

If we store the date of sign up in the user document we can always calculate the `isAdult` flag in an environment object somewhere in the view hierarchy, but all of the might need to happen in #52.